src/lib.rs

```diff
@@ -1153,12 +1155,6 @@ mod panic {
 
 struct LoggerGuard(&'static Log);
 
-impl Drop for LoggerGuard {
```

Comment:
This synchronization needs to stick around, since `Log` isn't an `unsafe` trait.